linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/bash/shared.sh

```diff
@@ -1,11 +1,7 @@
 # platform = multi_platform_wrlinux,multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
 
-# Find which files have incorrect hash {{{ "(not in /etc, because there are all system related config. files) " if product == "rhel6" }}}and then get files names
-{{%- if product == "rhel6" %}}
-files_with_incorrect_hash="$(rpm -Va | grep -E '^..5.* /(bin|sbin|lib|lib64|usr)/' | awk '{print $NF}' )"
-{{%- else %}}
+# Find which files have incorrect hash {{{ "(not in /etc, because there are all system related config. files) " }}} and then get files names
```

Comment:
this looks like an unwanted Jinja invocation, there was probably some collision in past